server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -104,23 +104,6 @@ def parser_newsenvelop(self, envelop_el):
         if element is not None:
             item['ingest_provider_sequence'] = element.text
 
-        # news_services CV
-        for element in envelop_el.findall('NewsService'):
-            if element is not None and element.get('FormalName'):
-                item.setdefault('subject', []).append({
-                    "name": element.get('FormalName'),
-                    "qcode": element.get('FormalName'),
-                    "scheme": "news_services"
-                })
-
-        for element in envelop_el.findall('NewsProduct'):
-            if element is not None and element.get('FormalName'):
-                item.setdefault('subject', []).append({
-                    "name": element.get('FormalName'),
-                    "qcode": element.get('FormalName'),
-                    "scheme": "news_products"
-                })
-
         element = envelop_el.find('Priority')
```

Comment:
Belga don't use product and service that get from external source. Instead of this, they will auto mapping the product and service follow key-works, catalog, ....
So we remove these code.